data_structures/skip_list.cpp

```diff
@@ -0,0 +1,197 @@
+/**
```

Comment:
Please see [`data_structures/trie_modern.cpp`](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/data_structures/trie_modern.cpp) as an example for implementation and documentation standards.

Reply:
☝️

Reply:
Should I return inner function comments?

Reply:
what does that mean?

Reply:
comments inside the function

Reply:
inline comments in codes are always a great idea whether it is mandatory or not 😄 

But, please, please.. check the links I have shared multiple times to format the docs. We are following doxygen standard - links are all provided multiple times above. 
Thanks.

Reply:
still not done.

Reply:
Hello, I was away. I corrected and add  parameters/return value comments.